test/parallel/test-fs-glob.mjs

```diff
@@ -300,10 +300,55 @@ const patterns = {
   ],
 };
 
+
+test('Glob should be an iterator', () => {
+  const pattern = Object.keys(patterns)[0];
```

Comment:
this seems like it's relying on the order of the keys. is there a reason it wouldn't just assert on everything in patterns?

Reply:
the order does not matter, just want to have glob pattern, replaced with a hard-coded pattern